portal/gatsby-node.js

```diff
@@ -1,6 +1,6 @@
 exports.onCreatePage = async ({ page, actions }) => {
     const { createPage, deletePage } = actions;
-    const componentPageMatch = page.path.match(/\/.*react.*\/documentation\/(.*)\//);
+    const componentPageMatch = page.path.match(/\/(.*(react|util).*)\/documentation\/(.*)\//);
```

Comment:
Nå trenger denne en liten fiks.
```suggestion
    const componentPageMatch = page.path.match(/\/.*react.*\/documentation\/(.*)\//);
    const utilsPageMatch = page.path.match(/\/.*util.*\/documentation\/(.*)\//);
```

og litt lengre ned:

```
if(utilsPageMatch){
   makePage(`/komponenter/${utilsPageMatch[0].toLowerCase()`});
}
```